app/src/main/res/values/strings.xml

```diff
@@ -209,6 +218,9 @@ Please report any problems you encounter via Github.</string>
     <!-- Setting title -->
     <string name="introducer">Introducer</string>
 
+    <!-- Setting title -->
+    <string name="device_pause">Pause Device</string>
```

Comment:
Can you call this pause_device, as the string below is delete_device, so this matches up.

Reply:
ok